Assets/Scripts/GUI/SketchbookPanel.cs

```diff
@@ -233,44 +238,16 @@ void SetVisibleSketchSet(SketchSetType type)
                 switch (m_CurrentSketchSet)
                 {
                     case SketchSetType.User:
-                        if (m_PanelText)
-                        {
-                            m_PanelText.text = m_PanelTextStandard;
-                        }
-                        if (m_PanelTextPro)
-                        {
-                            m_PanelTextPro.text = m_PanelTextStandard;
-                        }
+                        m_PanelTextPro.text = PanelTextStandard;
```

Comment:
Presumably this was because there used to be a difference between whether TMPro was used between mobile and standard sketchbook panels. I guess that's changed now, and we no longer use the old TextMeshes?

Reply:
Yep, most if not all TextMeshes are gone as they couldn't be directly localized